### PR TITLE
Add `latestUpdate` to `progress-bar`

### PR DESCRIPTION
**Pull request by Samq64**
### Changes

Adds the update tag to the progress bar addon with a temporary notice about the changes from 1.30.1.

### Reason for changes

A lot of users might have missed it since there was no update tag on the settings page.

### Tests

Tested on Chromium 110.

### CODE REVIEW

File: addons/progress-bar/addon.json [hunk 1]
```diff
@@ -10,6 +10,10 @@
       "link": "https://scratch.mit.edu/users/jbthepig/"
     }
   ],
+  "latestUpdate": {
+    "version": "1.31.0",
+    "temporaryNotice": "The progress bar has had a redisign since Scratch Addons 1.30.1."
```
Comment (WorldLanguages) [inline addons/progress-bar/addon.json:15]: Typo: "redisign"

Comment (WorldLanguages) [reply addons/progress-bar/addon.json:15]: And add "v" before the version

Comment (TheColaber) [reply addons/progress-bar/addon.json:15]: I don't think we normally even include the version in the notice tho. Maybe just exclude that part?

Comment (WorldLanguages) [reply addons/progress-bar/addon.json:15]: @TheColaber Well, it's not bad to clarify that the change isn't technically new.

Comment (DNin01) [reply addons/progress-bar/addon.json:15]: > And add "v" before the version

Clarification: Keep `"version": "1.31.0"` the same but add the letter "v" in the `temporaryNotice` description.

Comment (Samq64) [reply addons/progress-bar/addon.json:15]: Should I remove "Scratch Addons" when I add the V?

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**